src/ExpressReceiver.ts

```diff
@@ -187,27 +188,31 @@ export function verifySignatureAndParseBody(
       } else {
         stringBody = (await rawBody(req)).toString();
       }
-      const signature = req.headers['x-slack-signature'] as string;
-      const ts = Number(req.headers['x-slack-request-timestamp']);
 
-      try {
-        await verifyRequestSignature(signingSecret, stringBody, signature, ts);
-      } catch (e) {
-        return next(e);
-      }
+      const {
+        'x-slack-signature': signature,
+        'x-slack-request-timestamp': requestTimestamp,
+        'content-type': contentType,
+      } = req.headers;
+
+      await verifyRequestSignature(
+        signingSecret,
+        stringBody,
+        signature as string | undefined,
+        requestTimestamp as string | undefined,
+      );
```

Comment:
moved this out of its own try/catch, since we already have a try/catch set up which does exactly the same thing